dragonfly/server.go

```diff
@@ -217,6 +221,30 @@ func (server *Server) SetName(a ...interface{}) {
 	server.name.Store(fmt.Sprint(a...))
 }
 
+// JoinMessage is the message that appears when a player joins the server. Leave this empty to disable it.
+// %v is the placeholder for the username of the player
+func (server *Server) JoinMessage() string {
```

Comment:
Is there actually any use for this?

Reply:
The use is just to return what the current join/leave message is

Reply:
Yeah, but is there any reason anyone would like... Have any use for that?

Reply:
I don't know lol, I just added it in case someone somewhere would need it.